charts/kube2iam/_kube2iam/daemonset.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube2iam
+  labels:
+    app: kube2iam
+spec:
+  template:
+    metadata:
+      labels:
+        name: kube2iam
+    spec:
+      hostNetwork: true
+      containers:
+        - image: jtblin/kube2iam:latest
+          name: kube2iam
+          args:
+            - "--base-role-arn=arn:aws:iam::593291632749:role/"
```

Comment:
I've just realised that the files in this directory, `_kube2iam`, are temporary files I used while developing the chart, and should be removed - the actual template files used are in `templates`